README.md

```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# JS Capstone
+
+This is the Capstone project for the second module in the Microverse program. <br>
+The project was based on [Scryfall API](https://scryfall.com/docs/api).
+
```

Comment:
- [ **OPTIONAL** ] What do you think if you update your project title by giving your project a good name that almost relates to a short idea about the content? This will make your documentation looks professional. 
I also suggest doing the same on the description, by updating it in order to get a short idea about the content. **A good Title and description will help users know the content of the project even before they watch its live demo**. I know you can do that :muscle: :muscle:

Reply:
You're right, we'll make sure to implement on it before merging 😃